docs/operator-manual/declarative-setup.md

```diff
@@ -1100,27 +1104,54 @@ stringData:
     }
 ```
 
-## Helm Chart Repositories
+## Helm
 
-Non standard Helm Chart repositories have to be registered explicitly.
```

Comment:
Why is this not longer a requirement? Since which version of Argo CD?
I think I remember if I didn't register the public ECR (no auth required) to install karpenter from the OCI registry it didn't work.